src/lib/controllers/aic.rs

```diff
@@ -100,7 +109,9 @@ pub async fn update(
             info_and_incr!(
                 statsd,
                 LogKey::AicRecordUpdate,
-                aic_id = &aic_id.to_string().as_str(),
+                aic_id = updated.id,
```

Comment:
TODO: I would like to also add the cj_event_value here, but need to figure out how to get it to only show up if it was updated

Reply:
Move this check `if existing.cj_event_value == data.cj_id || data.cj_id == empty_cj_id()` to it's own separate check that then powers the if statement on L75 and a logging branch here.